server/startup/migrations/v171.js

```diff
@@ -0,0 +1,16 @@
+import { Migrations } from '../../../app/migrations/server';
+import { Settings } from '../../../app/models/server';
+
+Migrations.add({
+	version: 171,
+	up() {
+		const settings = [
+			'AutoLinker_UrlsRegExp',
+		];
+
+		Settings.remove({ _id: { $in: settings } });
```

Comment:
Please use `Settings.remove({ _id:  'AutoLinker_UrlsRegExp'});` since you are removing only one setting and remove the variable above.

Reply:
Thank you. Fixed it.